sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/QueuePropertiesExtensions.cs

```diff
@@ -51,28 +53,32 @@ public static XDocument Serialize(this QueueProperties description)
         /// <summary>
```

Comment:
nit:  These offer no value;  you may want to  remove them if you're not going to populate.